pom.xml

```diff
@@ -944,7 +944,7 @@
                                     <version>${maven.version.range}</version>
                                 </requireMavenVersion>
                                 <requireJavaVersion>
-                                    <version>${java.version}</version>
+                                    <version>11</version>
```

Comment:
Could we just keep the origin version here, otherwise we have to add `-Denforcer.skip=true` to do any other operations with Java 8.

Reply:
Yeah, we can. But It cuould confuse people that building with Java 8 will fail after upgrading antlr to 4.10.1. Anyway, we need to add a NOTE in README to mention that `Building from source` of shardingsphere must use `JDK 11` above. @terrymanu WDYT?

Reply:
I prefer to create `compile java version` and `release java version` properties to separate the different versions.